### PR TITLE
added simon game

### LINKED ISSUE #172: add simon game
The Simon Game is a classic memory game where you must follow a sequence of colors and sounds. Build using html,css and javascript.

The game starts with a single color and sound, and then progressively adds more to the sequence. Your task is to remember and repeat the sequence correctly. If you got the sequence incorrect, you will get Game Over and you can then restart the game again.

Simon - Google Chrome 10-10-2023 16_28_39

### PR DESCRIPTION
**Pull request by VishakhaSainani**
# Title and Issue number 

Title: Simon game

Issue No. : #172

Code Stack : html, css, javascript.

Close #172 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/simon-game/game.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+userClickedPattern=[];
+var started=false;
+var level=0;
+var buttonColours=["red", "blue", "green", "yellow"];
+var gamePattern=[];
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/game.js:5]: use const or let

File: native/machine-coding/simon-game/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Simon</title>
+  <link rel="stylesheet" href="styles.css">
+  <script src="../../helpers/navbar.js" type="module"></script>
+  <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/index.html:9]: no new fonts please

File: native/machine-coding/simon-game/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+</head>
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/index.html:10]: import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags

File: native/machine-coding/simon-game/index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+      </div>
+      <div type="button" id="blue" class="btn blue">
+
+      </div>
+
+    </div>
+
+  </div>
+  <script src="https://code.jquery.com/jquery-3.6.4.js" integrity="sha256-a9jBBRygX1Bh5lt8GZjXDzyOB+bWve9EiO7tROUtj/E=" crossorigin="anonymous"></script>
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/index.html:38]: we will not use jquery

File: native/machine-coding/simon-game/styles.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
+@import url(../../helpers/core.css);
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/styles.css:1]: this can be removed

File: native/machine-coding/simon-game/styles.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+body {
+  text-align: center;
+  background-color: #011F3F;
+}
+
+#level-title {
+  font-family: 'Press Start 2P', cursive;
```
Comment (sadanandpai) [inline native/machine-coding/simon-game/styles.css:9]: no new font family

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by VishakhaSainani** [comment]
Changes done , please review, Thankyou:)

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**